core/src/main/java/overflowdb/NodeDb.java

```diff
@@ -637,7 +637,7 @@ private final synchronized int storeAdjacentNode(Direction direction, String edg
     int strideSize = getStrideSize(edgeLabel);
 
     Object[] adjacentNodesWithEdgeProperties = adjacentNodes.nodesWithEdgeProperties;
-    int edgeOffsetLengthB2 = adjacentNodes.edgeOffsets.length() >> 1;
+    int edgeOffsetLengthB2 = adjacentNodes.getIntOffsetLen() >> 1;
```

Comment:
Hmm, I thought `edgeOffsets.length` was more helpful to the reader than `getIntOffsetLen`. 
Not a very clear winner here I guess, maybe something like `adjacentNodes.offsetLength()`?

Reply:
`edgeOffsets.length` was more helpful to the reader. However, `edgeOffsets` as an object doesn't exist anymore, so we can't ask it for its length.

But I'm happy to change the name, if you have any suggestions!

Reply:
Ok, renamed it